apps/server/src/modules/account/services/account.service.spec.ts

```diff
@@ -114,7 +114,7 @@ describe('AccountService', () => {
 	});
 
 	beforeEach(() => {
-		jest.useFakeTimers('modern');
+		jest.useFakeTimers();
```

Comment:
what is the different?

Reply:
none, see jest migration guides, not needed to specify that string anymore